tests/aspnet-mvc/OwinWebApp/Web.config

```diff
@@ -9,11 +9,6 @@
     <add key="webpages:Enabled" value="false"/>
     <add key="ClientValidationEnabled" value="true"/>
     <add key="UnobtrusiveJavaScriptEnabled" value="true"/>
-    <add key="ida:ClientId" value="4cb8c23b-0612-4b84-ae0f-03a0ff654e1f"/>
```

Comment:
This is also possible to keep these settings, but it's more complex to describe the certificates.
Can we comment these lines out (for the docs)

Reply:
yes. will update.